policies/coding-style.md

```diff
@@ -738,7 +738,30 @@ For example,
     }
 ```
 
-## Chapter 16: References
+## Chapter 16: Asserts
+
+We have 3 kind of asserts. The behaviour depends on being a debug or release build:
+| Function       | failure release | failure debug | success release | success debug |
+| -------------- | --------------- | ------------- | --------------- | ------------- |
+| assert         | not evaluated   | abort         | not evaluated   | nothing       |
+| ossl_assert    | returns 0       | abort         | returns 1       | returns 1     |
+| OPENSSL_assert | abort           | abort         | nothing         | nothing       |
+
+Use OPENSSL_assert() **only** in the following cases:
+- In the libraries when the global state of the software is corrupted and there is no way to recover it
+- In applications, test programs and fuzzers
+
+Use ossl_assert() in the libaries when the state can be recovered and an error can be returned. Example code:
```

Comment:
s/libaries/libraries/

Reply:
IMO minor edit that can be fixed when merging. If you agree please at least two OTC members put +1 here.

Reply:
+1

Reply:
+1

Reply:
+1 for trivial minor edit